README.md

```diff
@@ -31,3 +45,43 @@ Then run
 pm2 logs
 ```
 You must observe blocks being submitted.
+
+Docker:
+
+## Currently we have the following docker options:
+
+1. Rainbow Docker image containing rainbowup ready for running
+	- run the rainbowup docker image with a custom command
+2. A development docker compose setup (docker-compose-dev.yml)
```

Comment:
would be better to name it just docker-compose.yml, because local development happens more often than prod. and do every docker-compose command with `-f docker-compose-dev.yml` is lengthy

Reply:
I agree, on that. I just wanted to be more explicit, so people don't get confused by thinking the docker-compose.yml is the production one.

Reply:
Makes sense, but
docker-compose.yml
docker-compose-prod.yml

would be list together, due to their filename, on github-ui or a local file browser, so it's unlikely people would ignore the prod one though. And also, we'll have documentation on running prod, people need to read that anyway for a production deployment, there we can explicitly mention docker-compose-prod.yml

Reply:
Ok, will change it to docker-compose.yaml than. Makes sense to me :)